### ISSUE TITLE
Failed to download sources: module boost: The requested URL returned error: 403 Forbidden

### ISSUE THREAD

**Issue by superuser-miguel** [flathub/org.kde.krita#39]
```
Downloading https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Failed to download sources: module boost: The requested URL returned error: 403 Forbidden
```
Changed this to : 
```

Downloading https://boostorg.jfrog.io/artifactory/main/release/1.74.0/source/boost_1_74_0.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  104M  100  104M    0     0  1283k      0  0:01:23  0:01:23 --:--:-- 1535k
```

**Comment by hfiguiere** [comment]
You can submit a pull request. It is known that the boost download URL changed.

**Comment by hfiguiere** [comment]
Closed with #40
